simpleclient_spring_boot/src/test/java/io/prometheus/client/matchers/CustomMatchers.java

```diff
@@ -0,0 +1,43 @@
+package io.prometheus.client.matchers;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.core.IsCollectionContaining;
+
+/**
+ * @author <a href="http://stackoverflow.com/users/4483548/bretc">BretC</a>
```

Comment:
Since Feb-1-2016 all code posted on StackOverflow is licensed under MIT License. I'll append this notice.